drivers/did-io/src/types.d.ts

```diff
@@ -1,21 +1,41 @@
 type Driver = {};
 
-declare module 'did-io' {
-  function use(methodName: string, driver: Driver): void;
-  function register(options: any): any;
-  function get(options: any): any;
-
-  export = {
-    use,
-    register,
-    get,
-  };
+declare module '@digitalbazaar/did-io' {
```

Comment:
Can you help me understand these type definition files? Are they manually edited because the source libraries do not provide them? Why can we not import or generate them?

Reply:
They aren't provided (not anywhere I could find)... we can generate it ?